common/changes/@uifabric/fluent-theme/miwhea-fluent-theme-remove-blues_2019-02-13-18-14.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/fluent-theme",
+      "comment": "Remove unnecessary theme colors",
+      "type": "patch"
```

Comment:
From a package maintainer and consumer perspective, would this change be considered `minor` rather than `patch` since the API contract seems to be changing?

Reply:
It should not be considered minor as nothing changed. Consumers will still be able to use `palette.theme...` colors as before and they even map to the same values as the ComunicationColors. So from the perspective of the API nothing changed.